### DISCUSSION TITLE
Privacy Manifest file not included in Build directory? (Carthage)

### DISCUSSION THREAD

**Discussion by NewFieldForMe** [krzyzanowskim/CryptoSwift#1039] [Q&A]
Hi,
I am trying to install the latest version(v1.8.1) using Carthage.
But I could not find the `PrivacyInfo.xcprivacy` file in the `Build` directory.

I try below step.

1. Make `Cartfile` and describe below.
```
github "krzyzanowskim/CryptoSwift" == 1.8.1
```

2. I executed the following command.
```
$ carthage update --platform iOS --use-xcframeworks CryptoSwift
```

3. I executed the following command.
```
$ find . -name '*privacy*'
```

result: 
```
./Carthage/Checkouts/CryptoSwift/Sources/CryptoSwift/PrivacyInfo.xcprivacy
```

There is a `PrivacyInfo.xcprivacy` in the `Source` directory, but no `PrivacyInfo.xcprivacy` in the `Build` directory.
I think `PrivacyInfo.xcprivacy` should be included in `CryptoSwift.xcframework` in the `Build` directory.

What do you think?

Thank you for your consideration.

**Comment by CH3COOH** [comment]
Hello!
I am using CryptoSwift and Kingfisher in my application. Both are installed via Carthage.

As you mentioned, the CryptoSwift.xcframework built with Carthage did not contain PrivacyInfo.xcprivacy. On the other hand, Kingfisher contained PrivacyInfo.xcprivacy.

After a little investigation, we may have forgotten the process of copying the resource.

* Kingfisher.
    * https://github.com/onevcat/Kingfisher/pull/2156/files
* CryptoSwift.
    * https://github.com/krzyzanowskim/CryptoSwift/commit/d7de7d73e8c4781f725a024419c9f61bee4f70a1

Thanks.
